src/Modal.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import cx from 'classnames';
+import Icon from './Icon';
+import Button from './Button';
+
+/**
+ * SQ2 Modal component
+ * @see {@link http://meetup.github.io/sassquatch2/views.html#modals}
+ * @module Modal
+ */
+class Modal extends React.Component {
+	constructor(props){
+		super(props);
+		this.onDismiss = this.onDismiss.bind(this);
+		this.handleKeyDown = this.handleKeyDown.bind(this);
+	}
+
+	onDismiss(e) {
+		e.stopPropagation();
+
+		if (this.props.onDismiss) {
+			this.props.onDismiss(e);
+		}
+	}
+
+	handleKeyDown(e) {
```

Comment:
excellent - now you just need to make sure the modal elements gets 'focus' as soon as the modal opens. I think you'll want to do that in `componentDidMount`, depending on how it's used